### PR TITLE
perf(SuggestionsList) Remove Callout, support right-alignment for RTL use-case

### PR DESCRIPTION
**Pull request by richardtagger**
See https://github.com/Microsoft/YamUI/issues/243

We don't use Callout's features such as beak, dynamic positioning or animation. Coupled with the poor performance issues, I've removed the use of it and we can revisit if the performance improves or we need to use more features from Callout

### CODE REVIEW

**Comment by richardtagger** [comment]
@unindented @swese44 @adrianmg

File: src/components/SuggestionsList/SuggestionsList.tsx [hunk 1]
```diff
@@ -5,16 +5,18 @@ import { NestableBaseComponentProps } from '../../util/BaseComponent/props';
...
 import './SuggestionsList.css';
-import { IPoint } from 'office-ui-fabric-react/lib/utilities/positioning';
 
 export interface SuggestionsListProps extends NestableBaseComponentProps {
   /**
-   * The target that the Callout should try to position itself based on.
+   * The left offset position in pixels
    */
-  target: HTMLElement | IPoint;
+  left: number;
```
Comment (unindented) [inline src/components/SuggestionsList/SuggestionsList.tsx]: This is not RTL-friendly.

Comment (richardtagger) [reply src/components/SuggestionsList/SuggestionsList.tsx]: you're right :(. It's no less RTL friendly than our current stuff but I'll have a think about how we can solve it

**Comment by unindented** [comment]
The hardcoded `left` is a blocker. Other than that it looks good to me.

**Comment by richardtagger** [comment]
Here's a thing https://github.com/Microsoft/YamUI/pull/244/commits/2d35ea9c90644169e5f55bb5c9bf30d321b12b30

**Comment by richardtagger** [comment]
@adrianmg @mamuso the screenshots changed too so I'll need your LGTM please

**Comment by unindented** [comment]
@richardtagger do we need the absolute positioning to live inside this component? Couldn't the consumer position it however it wants?

**Comment by richardtagger** [comment]
gonna close this PR and put together something that is in that direction

**Pull request closed without merging**